src/layouts/Header.tsx

```diff
@@ -108,6 +117,8 @@ class Header extends React.Component<IProps, null> {
         this.props.disableTrainingTooltipsShowAll();
       };
     }
+
+    this.setState({ alchemyVersion: (await axios("https://raw.githubusercontent.com/daostack/alchemy/master-2/package.json"))?.data?.version ?? "Not found" });
```

Comment:
lets avoid adding another dependency (GitHub) .  
take the version during build time from package.json.

Reply:
Done